recipe/meta.yaml

```diff
@@ -9,26 +9,36 @@ package:
 source:
   fn: {{ filename }}
   url: https://github.com/rdkit/rdkit/archive/{{ filename }}
-  sha256: 38e6fb9f063b6132310f17e654f2c4350876f9164b0a17b49fe3df7d0555a744
+  sha256: edd30682cc3031cf3f2b1a400f453629db332a1018f355cd3f7ff76b2f7f5398
 
 build:
   number: 0
-  skip: true  # [aarch64]
+  # We skip s390x because it's not supported, and win because of boost weirdness.
+  # win SHOULD work though, and this recipe does build with boost from CF so it's
```

Comment:
Could this recipe be helpful https://github.com/rdkit/conda-rdkit/tree/master/rdkit ?

Reply:
Hmm I already tried it and our recipe is practically the same, so I have no clue what's going on. I will have to investigate it in detail when I get some time. In essence, CMake is unable to find `boost_python` even though it exists in our `libboost` and `py-boost` packages, and curiously it only happens on `win`.